group_vars/all

```diff
@@ -23,6 +23,7 @@ proxy_env:
   HTTPS_PROXY: ''
 http_proxy: ''
 prometheus_enabled: true
+promdash_enabled: true
```

Comment:
The flags on global works at the playbook level whereas yours is at the role level, we should probably try to be consistent